axelrod/strategies/dbs.py

```diff
@@ -0,0 +1,354 @@
+from axelrod.actions import Actions, Action
+from axelrod.player import Player
+
+
+C, D = Actions.C, Actions.D
+
+def action_to_int(action):
+    return (1 if action==C else 0)
+
+
+class DBS(Player):
+    """
+    Desired Belief Strategy as described in:
+    Accident or Intention: That Is the Question (in the Noisy Iterated Prisoner's Dilemma) by Tsz-Chiu Au and Dana Nau from University of Maryland
```

Comment:
Thanks for including the reference. We have a citation file and a specific format we prefer, can you try to convert? We can help...